### DISCUSSION TITLE
Jove's shield sprite is messed up

### DISCUSSION THREAD

**Discussion by EebleSheembl** [FakeFishGames/Barotrauma#14420] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/4c75da09-320b-4053-a356-e1ec7c97ae60)

### Reproduction steps

Go into creature editor
Go onto Jove

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/39524897a70792f46c30bef6caf156f065378762 (commit in our private repo)
